test/sds/SDS_Regression.ipynb

```diff
@@ -1046,7 +1045,160 @@
+    "    print(f'HOSS/MaskFill chained service is not configured for environment: \"{harmony_environment}\" - skipping test.')\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "72eeff5d-0893-4937-bcd9-a187e548b961",
+   "metadata": {},
+   "source": [
+    "## Spatial subsetting of projected grid.\n",
+    "\n",
+    "This request uses the chained service that combines HOSS with MaskFill to offer bounding box spatial subsetting of coordinate projected gridded data hosted in OPeNDAP.\n",
+    "\n",
+    "The request will use the [ABoVE Tundra Vegetation Photosynthesis and Respiration Model (TVPRM) Simulated Net Ecosystem Exchange collection](https://cmr.uat.earthdata.nasa.gov/search/concepts/C1245804308-EEDTEST.html). This collection uses an Albers Conical Equal Area projection. \n",
+    "\n",
+    "The request also uses a temporal subset to limit the size of the result.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4657e58b-8153-4f81-a794-3f8f8a1620b1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hoss_projected_non_prod_information = {\n",
+    "    'collection': Collection(id='C1245804308-EEDTEST'),\n",
+    "    'bbox': BBox(w=-160, s=68, e=-150, n=70),\n",
+    "    'granule_id': 'G1245804356-EEDTEST',\n",
+    "    'temporal_range': {'start': datetime(2008, 7, 2, 0, 0, 0),\n",
+    "                 'stop': datetime(2008, 7, 2, 1, 0, 0)}\n",
+    "\n",
+    "}\n",
+    "\n",
+    "hoss_projected_env = {Environment.LOCAL: hoss_projected_non_prod_information,\n",
+    "                     Environment.SIT: hoss_projected_non_prod_information,\n",
+    "                     Environment.UAT: hoss_projected_non_prod_information}\n",
+    "\n",
+    "if harmony_environment in hoss_projected_env:\n",
+    "    hoss_projected_info = hoss_projected_env[harmony_environment]\n",
+    "else:\n",
+    "    hoss_projected_info = None"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "64f96581-a7bb-4221-ab8c-b4a81aa15a71",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if hoss_projected_info is not None:\n",
+    "    hoss_projected_reference_filename = 'reference_images/hoss_projected_north_slope_reference_one.nc4'\n",
+    "    hoss_projected_filename = 'hoss_projected_north_slope.nc4'\n",
+    "    hoss_projected_request = Request(collection=hoss_projected_info['collection'],\n",
+    "                                     granule_id=hoss_projected_info['granule_id'],\n",
+    "                                     spatial=hoss_projected_info['bbox'],\n",
+    "                                     variables=['NEE', 'lat', 'lon'],\n",
+    "                                     temporal=hoss_projected_info['temporal_range'])\n",
+    "    submit_and_download(harmony_client, hoss_projected_request, hoss_projected_filename)\n",
+    "    assert exists(hoss_projected_filename), 'Unsuccessful HOSS spatial subset request.'\n",
```

Comment:
The other tests finish up by printing out a green bit of text that says the test was successful. It's not a big deal at all, but I thought it was a nice visual way to see that the test succeeded maybe. (Although in this case that text would likely come after the comparison to the reference image, which is the most important part of this test)

Reply:
I moved the comparison under the images and added a print_success statement as the last part of this test.